react/components/FilterBar/FilterTag.js

```diff
@@ -339,6 +341,7 @@ FilterTag.propTypes = {
   newFilterLabel: PropTypes.string,
   device: PropTypes.string,
   isMobile: PropTypes.bool,
+  noOptionsMessage: PropTypes.func,
```

Comment:
Why a function? How would that be used?

Reply:
It receives the value and returns the message to be displayed. This is just forwarded to the library used for the select